spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+
+RSpec.describe 'User', type: :feature do
+  before(:each) do
+    # @user1 = User.create(name: 'Lebron', photo: 'https://lebronlinktophoto.jpg', bio: 'Make a diffrence',
+    #                      posts_counter: 0)

```

Comment:
- [ ] It's considered a good practice to have a readable code that is clean of any commented (unused) code. Please remove this commented code.